src/app/components/navigation/navigation.css

```diff
@@ -0,0 +1,80 @@
+.search-box {
+  flex: 1;
+  text-align: center;
+  display: flex;
+  justify-content: center;
```

Comment:
One thing i did with the quiz pages was that I made just the CSS class for flex-row and flex-column so that I can just apply that class to whichever component you intend to make flex. and I did the same with justify content , I made the class called justify-center and keep adding that same class to different elements and that way your code has little less repeating, I saw the same thing with team d2's MA Hetav in his code.

Reply:
sure dipen. I will make sure to use more utilities classes in second iteration